types/jsonb.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+type Jsonb struct {
```

Comment:
Is this a workaround to keep a simpler JSON type and not have to worry about merging the handling logic for the new JSON gorm type? https://github.com/go-gorm/datatypes/blob/master/json.go

Reply:
Indeed, that's better, changing